src/app/environments/environment.interface.ts

```diff
@@ -1,4 +1,5 @@
 export interface EnvironmentInterface {
+  production: boolean;
```

Comment:
boolean am besten immer mit einem Verb anfangen lassen wie `is` oder `has`.
Damit sind diese eindeutiger in der Interpretation.
```suggestion
  isProduction: boolean;
```